cmd/ocm/edit/ingress/cmd.go

```diff
@@ -55,6 +67,12 @@ const (
 	namespaceOwnershipPolicyFlag  = "namespace-ownership-policy"
 	clusterRoutesHostnameFlag     = "cluster-routes-hostname"
 	clusterRoutesTlsSecretRefFlag = "cluster-routes-tls-secret-ref"
+	componentRoutesFlag           = "component-routes"
+
+	amountOfParametersPerComponentRoute = 2
```

Comment:
perhaps we could avoid this by adding `hostnameParameter `  and `tlsSecretRefParameter` to its own slice

Reply:
actually you are already doing this in `expectedParameters` - can we just use that instead?

Reply:
Oh true, I had missed that. Let's adjust!

Reply:
I'll still need const though, it just so happened that the parameters currently match the [parsed component length](https://github.com/openshift-online/ocm-cli/pull/606/files#diff-e49b4de7ddb1004c3a10c0f5c3867e9c133741d618742f7f14462b1173497232R357), but I would rather keep them separate for future proofing. But I went ahead and renamed for better intention and used the [expectedParams](https://github.com/openshift-online/ocm-cli/pull/606/files#diff-e49b4de7ddb1004c3a10c0f5c3867e9c133741d618742f7f14462b1173497232R373) for the actual param length